### ISSUE TITLE
Various block-level elements are 'outdented' and it looks ugly

### ISSUE THREAD

**Issue by shimizukawa** [bitprophet/alabaster#32]
I wrote this docstring:

```
def bar(age):
    """
    this is bar function

    >>> import foo
    >>> foo.bar(10)
    10

    This function just return age.

    :param int age: your age
    :return: age
    :rtype: int
    """
    return age
```

and built with using 'classic' theme, I got this image:
![image](https://cloud.githubusercontent.com/assets/151623/6769407/503a6148-d0dd-11e4-8b05-d7cb56aea3c6.png)

I git this image with using 'alabaster' theme:
![image](https://cloud.githubusercontent.com/assets/151623/6769408/5789a788-d0dd-11e4-811e-1b08031def94.png)

It would be better to understand the document if gray box for doctest lines doesn't protrude outside the docstring content area.

**Comment by bitprophet** [comment]
Yea, we have this issue with other block-level elements as well (ones which are also code blocks, IIRC). This looked like an intentional style for the source styles from armin/kenneth, but I've never been a huge fan and intended to change it sometime.

**Comment by bitprophet** [comment]
Changed title to reflect that this affects various block level elements, not just code blocks. (E.g. admonitions also, per #85).

**Comment by inducer** [comment]
I'm not sure if it's helpful, but the code blocks in my [example demonstrating the issue for admonitions](https://documen.tician.de/pyopencl/runtime_memory.html#pyopencl.SVM) look OK, whereas the admonitions are off in the way described. FWIW, I'm using `::<newline>`-style code blocks, not `>>>`-style ones.

**Comment by bitprophet** [comment]
Yea those may spit out different elements, not sure, I don't use the python-shell style ones myself.

Either way, when I tackle this I'm going to try finding all rules like the one you indicated and make sure they're all fixed.

**Comment by bitprophet** [comment]
Took a couple screenshots of my own to highlight an amusing corner case of this issue, which is making it even more pressing for me personally. (As-is, it's ugly, but doesn't necessarily _look_ broken...but when you put a code block within an already-fixed note block, it _does_ look broken.)

Normal example of how code blocks are dedented:

(TK, once S3 finishes shitting itself...)

What happens when you use a code block within a note block:

(ditto...)

----

The fix appears to be pretty simple, simply nuke [this CSS rule](https://github.com/bitprophet/alabaster/blob/48bfef380ddfe11d3e16322b77575997873386dc/alabaster/static/alabaster.css_t#L501-L503) and they look much better. (Removing just the negative margin and leaving the padding understandably makes it _too_ indented.)

Similarly, nuking https://github.com/bitprophet/alabaster/blob/48bfef380ddfe11d3e16322b77575997873386dc/alabaster/static/alabaster.css_t#L279-L282 seems to fix the admonition variant of the problem.

And I see no other `margin-left: -NN` rules so that is hopefully it.

**Comment by bitprophet** [comment]
For completion, screenies of how the above looks with the fix in. Just-a-code-block:

![fixed-code-block](https://cloud.githubusercontent.com/assets/6088/23440263/ec1494a0-fdcf-11e6-8df3-2db82f8add06.png)

Nested code block in note:

![fixed-nested](https://cloud.githubusercontent.com/assets/6088/23440274/f24d8318-fdcf-11e6-89e6-d1ada7c1055c.png)

This has been released in 0.7.10 so will be picked up by fresh Sphinx installs etc.

**Comment by shimizukawa** [comment]
Great! Thanks!!!
